tests/src/Functional/ContactsDashboardTest.php

```diff
@@ -270,16 +270,19 @@ public function testViewDashboard() {
   public static function sortContacts(UserInterface $a, UserInterface $b) {
     // First sort by roles.
     $a_roles = $a->getRoles();
```

Comment:
Let's put an `@todo` in to replace this with something better for sorting - create and link to an issue for it